src/components/heading/heading-test.stories.mdx

```diff
@@ -69,8 +69,6 @@ export const HeadingStory = ({
       helpLinkSpecialCharacters: undefined,
       backLink: "",
       backLinkSpecialCharacters: undefined,
-      divider: Heading.defaultProps.divider,
-      separator: Heading.defaultProps.separator,
```

Comment:
I think you should have left these two args but with explicit values instead of references:

`divider: true,
separator: false,`